elevate-survey/constants/routes.js

```diff
@@ -70,6 +80,236 @@ module.exports = {
 				type: 'GET'
 			},
 		},
+		{
+			sourceRoute: '/samiksha/v1/solutions/create',
+			type: 'POST',
+			inSequence: false,
+			orchestrated: false,
+			targetRoute: {
+				path: '/samiksha/v1/solutions/create',
+				type: 'POST'
+			},
+		},
+		{
+			sourceRoute: '/samiksha/v1/solutions/details/:id',
+			type: 'POST',
+			inSequence: false,
+			orchestrated: false,
+			targetRoute: {
+				path: '/samiksha/v1/solutions/details/:id',
+				type: 'POST'
+			},
+		},
+		{
+			sourceRoute: '/samiksha/v1/solutions/verifyLink/:id',
+			type: 'POST',
+			inSequence: false,
+			orchestrated: false,
+			targetRoute: {
+				path: '/samiksha/v1/solutions/verifyLink/:id',
+				type: 'POST'
+			},
+		},
+		{
+			sourceRoute: '/samiksha/v1/solutions/fetchLink/:id',
+			type: 'GET',
+			inSequence: false,
+			orchestrated: false,
+			targetRoute: {
+				path: '/samiksha/v1/solutions/fetchLink/:id',
+				type: 'GET'
+			},
+		},
+		{
+			sourceRoute: '/samiksha/v1/solutions/update/:id',
+			type: 'POST',
+			inSequence: false,
+			orchestrated: false,
+			targetRoute: {
+				path: '/samiksha/v1/solutions/update/:id',
+				type: 'POST'
+			},
+		},
+		{
+			sourceRoute: '/samiksha/v1/solutions/getDetails/:id',
+			type: 'GET',
+			inSequence: false,
+			orchestrated: false,
+			targetRoute: {
+				path: '/samiksha/v1/solutions/getDetails/:id',
+				type: 'GET'
+			},
+		},
+		{
+			sourceRoute: '/samiksha/v1/solutions/list',
+			type: 'GET',
+			inSequence: false,
+			orchestrated: false,
+			targetRoute: {
+				path: '/samiksha/v1/solutions/list',
+				type: 'GET'
+			},
+		},
+		{
+			sourceRoute: '/samiksha/v1/solutions/isTargetedBasedOnUserProfile/:id',
+			type: 'GET',
+			inSequence: false,
+			orchestrated: false,
+			targetRoute: {
+				path: '/samiksha/v1/solutions/isTargetedBasedOnUserProfile/:id',
+				type: 'GET'
+			},
+		},
+		{
+			sourceRoute: '/samiksha/v1/solutions/detailsBasedOnRoleAndLocation/:id',
+			type: 'GET',
+			inSequence: false,
+			orchestrated: false,
+			targetRoute: {
+				path: '/samiksha/v1/solutions/detailsBasedOnRoleAndLocation/:id',
+				type: 'GET'
+			},
+		},
+		{
+			sourceRoute: '/samiksha/v1/solutions/forUserRoleAndLocation',
+			type: 'GET',
+			inSequence: false,
+			orchestrated: false,
+			targetRoute: {
+				path: '/samiksha/v1/solutions/forUserRoleAndLocation',
+				type: 'GET'
+			},
+		},
+		{
+			sourceRoute: '/samiksha/v1/solutions/addEntitiesInScope/:id',
+			type: 'POST',
+			inSequence: false,
+			orchestrated: false,
+			targetRoute: {
+				path: '/samiksha/v1/solutions/addEntitiesInScope/:id',
+				type: 'POST'
+			},
+		},
+		{
+			sourceRoute: '/samiksha/v1/solutions/addRolesInScope/:id',
+			type: 'POST',
+			inSequence: false,
+			orchestrated: false,
+			targetRoute: {
+				path: '/samiksha/v1/solutions/addRolesInScope/:id',
+				type: 'POST'
+			},
+		},
+		{
```

Comment:
@praveenKDass Please check if you have to add one more config for all endpoints where we pass /:id such that the other configuration doesn't have :id
eg : /samiksha/v1/solutions/addRolesInScope

Reply:
Config all the endpoits which has :id @VISHNUDAS-tunerlabs